src/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -147,6 +147,14 @@
     <Compile Include="System\Net\Http\Managed\ManagedHandler.cs" />
     <Compile Include="System\Net\Http\Managed\RawConnectionStream.cs" />
   </ItemGroup>
+  <!-- Managed HttpClientHandler platform parts -->
+    <ItemGroup Condition=" '$(TargetsUnix)' == 'true' ">
+      <Compile Include="System\Net\Http\Managed\HttpProxyConnectionHandler.Unix.cs" />
+      <Compile Include="System\Net\Http\Managed\HttpEnvironmentProxy.cs"/>
```

Comment:
Nit: The existing lines have a space before `/>`. Applies to the line below for windows as well.